### ISSUE TITLE
After account activation, users should be taken to account preferences by default

### ISSUE THREAD

**Issue by khalwat** [craftcms/cms#7809]
Spurred on by https://github.com/craftcms/cms/issues/7790 in which it is stated that accessibility features will be enabled by default.

The fact is that most people will not need or want focus outlines and other accessibility features, and they can be a detriment to UX in the form of visual clutter.

However, we also do not want to make it difficult for people to who _do_ need or want focus outlines or other accessibility features to be able to find them.

I think this is a generic problem, though, and not related to accessibility, that can be solved by taking people right to their account preferences page by default after activating a new account, where they can enable any features that they need or want.

This is actually analogous to what Apple has done in Big Sur. Accessibility features are all **off** by default, but as of Big Sur, a dialog appears when you first set up your computer to let you enable the accessibility features you need.

Since no on/off default will be right for all organizations, as part of this, provide a "new user settings template" that the site admin can control, to allow them to default various new user settings in a way that make sense for their use-case.

IMO, set defaults that make sense for the majority of users, but make it very easy (and not hidden) for people to change things as they need to. But others may disagree, thus having "new user settings" that site admins can adjust (perhaps via Group settings?) will address that nicely.

I think this is the way to go.

**Comment by lindseydiloreto** [comment]
I believe that accessibility features should be **on** by default, but more importantly, I agree that this should be asked a config setting from the very beginning, when the site is being installed.

**Comment by michtio** [comment]
I do agree with you on this @khalwat - off by default, but when a first login occurs in a very accessible way presenting the options to the user.

@lindseydiloreto even tho you believe it should be "on" by default, I would disagree that this is a setting when Craft is installed, as like that you'd enforce the entire site to handle with those `a11y` cases, if you're client only has 1 or 2 authors I could say yes, but when you handle a site that has 100+ authors, and only 1 of them is helped by the accessibility features, I would say, don't annoy the other 99 ... 

Accessibility, with a big emphasis on importance, you still need to be careful you don't create annoyance by the people not needing/wanting it.

That's why I'm 100% supporting Andrew's idea as introducing this as a setting on an individual level, which is nicely integrated, not hidden, and easily reached. ( Maybe even it's own element in the sidebar, or anything ). 

The only thing I'm opposed against is "hiding it" "turning things on by default" etc ... as hiding it is bad for the people need accessibility and enforcing it will frustrate/annoy the people who don't want it. So the solution as stated in this issue is for me the perfect middle ground :)

**Comment by lindseydiloreto** [comment]
> ... if your client only has 1 or 2 authors I could say yes, but when you handle a site that has 100+ authors, and only 1 of them is helped by the accessibility features, I would say, don't annoy the other 99.

This is the fundamental problem. You've actually made the site **accessible** to a whole new person, while only _annoying_ the others. It makes sense as a mathematical "greater good", but doesn't try to make the technology available to as many people as possible.

Personally, I agree with you... for me, the accessibility features are overkill (and slightly annoying). And since our team has no special accessibility needs, we'll simply be disabling it with a config setting. 🙂

**Comment by khalwat** [comment]
Well, I think it can be both right? We have no desire to exclude anyone, and the best way to do that is to ensure that any special accessibility features are added in a non-invasive way. That gets people behind the accessibility features.

I'm not sure that annoying 99 people by default to accommodate 1 by default is a great tradeoff, but I also don't think you have to make that tradeoff.

Most accessibility features are entirely transparent. They aren't apparent to anyone until they need them, and they aren't even noticed by the people who don't need them. You just have to take the time and diligence to add them.

I think a considered approach can ensure that things like focus outlines are also implemented in a similar manner. e.g.: https://github.com/craftcms/cms/issues/7805#issuecomment-821235537
